scripts/sign.py

```diff
@@ -69,7 +69,7 @@ def get_args(logger):
                         type=uuid_parse, help='String UUID of the TA')
     parser.add_argument('--version', type=int_parse, default=0,
                         help='Decimal version value for the TA, defaults to 0')
-    parser.add_argument('--key', required=True,
+    parser.add_argument('--key', required=False,
```

Comment:
`with open(args.key, 'rb') as f:` below will not like when key is empty I think ;-)

And `args.inf` has the same problem it seems.

Reply:
hmm, i see something a bit ugly. `--digest` needs `--key` to an obviously a dummy key but of same family&size to get key size value for signed header info.

Ok, currently --key and --inf are `Required` . I'll fixup my commit.